packages/migrate/sources/ghost.js

```diff
@@ -0,0 +1,170 @@
+                try {
+                    ctx.result = await ghostAPI.process.all(ctx);
+                    await ctx.fileCache.writeTmpFile(ctx.result, 'gh-processed-data.json');
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Build Link Map',
+            task: async (ctx) => {
+                // 3. Create a map of all known links for use later
+                try {
+                    ctx.linkFixer.buildMap(ctx);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Format data as Ghost JSON',
+            task: (ctx) => {
+                // 4. Format the data as a valid Ghost JSON file
+                try {
+                    ctx.result = mgJSON.toGhostJSON(ctx.result, ctx.options);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Fetch images via ImageSraper',
+            task: async (ctx) => {
+                // 5. Pass the JSON file through the image scraper
+                let tasks = ctx.imageScraper.fetch(ctx);
+                return makeTaskRunner(tasks, options);
+            },
+            skip: () => ['img'].indexOf(options.scrape) < 0
+        },
+        {
+            title: 'Update links in content via LinkFixer',
+            task: async (ctx, task) => {
+                // 6. Process the content looking for known links, and update them to new links
+                let tasks = ctx.linkFixer.fix(ctx, task);
+                return makeTaskRunner(tasks, options);
+            }
+        },
+        {
+            title: 'Write Ghost import JSON File',
+            task: async (ctx) => {
+                // 7. Write a valid Ghost import zip
+                try {
+                    await ctx.fileCache.writeGhostImportFile(ctx.result);
+                    await ctx.fileCache.writeErrorJSONFile(ctx.errors);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        },
+        {
+            title: 'Write Ghost import zip',
+            skip: () => !options.zip,
+            task: async (ctx) => {
+                // 8. Write a valid Ghost import zip
+                try {
+                    ctx.outputFile = fsUtils.zip.write(process.cwd(), ctx.fileCache.zipDir, ctx.fileCache.defaultZipFileName);
+                } catch (error) {
+                    ctx.errors.push(error);
+                    throw error;
+                }
+            }
+        }
+    ];
+};
+
+module.exports.getTaskRunner = (options) => {
+    let tasks = [];
+
+    if (options.info) {
```

Comment:
We should probably add `info` as an option to the commands (it works fine tho, just tested it 😉 )